rotkehlchen/chain/evm/decoding/interfaces.py

```diff
@@ -53,6 +53,10 @@ def enricher_rules(self) -> list[Callable]:  # pylint: disable=no-self-use
         """
         return []
 
+    def post_decoding_rules(self) -> list[Callable]:
+        """Subclasses may implement this to add post processing of the decoded events"""
```

Comment:
```suggestion
        """Subclasses may implement this to add post processing of the decoded events. This will run after the normal decoding step and will only process decoded history events"""
```